insights/tests/combiners/test_cloud_provider.py

```diff
@@ -621,6 +628,15 @@ def test_rpm_azure():
     assert ret.long_name == 'Microsoft Azure'
 
 
+def test_rpm_oracle():
+    irpms = IRPMS(context_wrap(RPMS_ORACLE))
+    dmi = DMIDecode(context_wrap(DMIDECODE_BARE_METAL))
+    ret = CloudProvider(irpms, dmi, None, None)
+    assert ret.cloud_provider == CloudProvider.ORACLE
+    assert ret.cp_rpms.get(CloudProvider.ORACLE)[0] == 'oracle-cloud-agent-2.2.18-1.fc40'
+    assert ret.long_name == 'Oracle Cloud'
```

Comment:
The pytest failed for this assert.  Looks like a typo here, `_LONG_NAME = 'Oracle Cloud Infrastructure'`.